### PR TITLE
remove JnaPtyExecutor as it doesn't work; add tests for failed native

### PR DESCRIPTION
**Pull request by segrey**

### CODE REVIEW

**Comment by traff** [comment]
Test is failing, could you please check it out.

File: test/com/pty4j/FailedToLoadNativePtyLibTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+public class FailedToLoadNativePtyLibTest extends TestCase {
+  public void testErrorMessage() {
+    String[] cmd = TestUtil.getJavaCommand(RepeatTextWithTimeout.class, "2", "1000", "Hello, World");
+    boolean failed = false;
+    try {
+      PtyProcess.exec(cmd);
+    }
+    catch (IOException e) {
+      failed = true;
```
Comment (traff) [inline test/com/pty4j/FailedToLoadNativePtyLibTest.java]: It would be nicer to use JUnit4 and syntax like: 
@Test(expected = IOException.class)

**Comment by segrey** [comment]
Test falure reproduced when FailedToLoadNativePtyLibTest runs after PtyTest. Fixed by https://github.com/traff/pty4j/pull/51/commits/90d623d0eb5fc05b884917279b5704bd2510cd49

**Comment by traff** [comment]
Thanks!

**Pull request merged**